### PR TITLE
Fix special token cleanup when using language models

### PR DESCRIPTION
**Pull request by dakshvar22**
**Proposed changes**:
- fixes https://github.com/RasaHQ/rasa/issues/5353

**Status (please check what you already did)**:
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py [hunk 1]
```diff
@@ -2,6 +2,23 @@
 import numpy as np
 
 
+def cleanup_tokens(
+    token_ids_string: List[Tuple[int, Text]], delimiter: Text
+) -> Tuple[List[int], List[Text]]:
+    """Utility method to apply specific delimiter based cleanup on list of tokens"""
```
Comment (tabergma) [inline rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py]: We should use the full docstring symtax, e.g. including `Args` and `Return`. But we also have another issue for that.

File: rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+    token_ids_string = [
+        (id, string.replace(delimiter, "")) for id, string in token_ids_string
+    ]
+
+    # remove empty strings
+    token_ids_string = [(id, string) for id, string in token_ids_string if string]
+
+    # return as individual token ids and token strings
+    token_ids, token_strings = zip(*token_ids_string)
+    return token_ids, token_strings
```
Comment (tabergma) [inline rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py]:
```suggestion
    return zip(*token_ids_string)
```

Comment (dakshvar22) [reply rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py]: @tabergma Get a type error if I do that, that's why had to first unpack it and then return.

Comment (tabergma) [reply rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py]: ok, thanks for the explanation, then I guess it is fine

File: rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py (hunk 1 above)
Comment (dakshvar22) [reply rasa/nlu/utils/hugging_face/transformers_pre_post_processors.py]: Thanks for pointing it out. Will address it as part of that issue.

**Pull request merged**